managed/src/test/java/com/yugabyte/yw/common/TemplateManagerTest.java

```diff
@@ -204,7 +205,7 @@ public void testTemplateCommandError() {
     expectedCommand.add("prometheus");
     when(shellProcessHandler.run(eq(expectedCommand), eq(new HashMap<>()), anyString()))
         .thenReturn(ShellResponse.create(1, "foobar"));
-    expectedException.expect(RuntimeException.class);
+    expectedException.expect(YWServiceException.class);
     expectedException.expectMessage("YBCloud command instance (template) failed to execute.");
```

Comment:
The issue is that structurred json YWException uses null message. 

Can you do following change:
```
  public YWServiceException(int httpStatus, String userVisibleMessage) {
    this(httpStatus, userVisibleMessage, null);
  }

  public YWServiceException(int httpStatus, JsonNode errJson) {
-    this(httpStatus, null, errJson);
+    this(httpStatus, "errorJson: " + errJson.toString(), errJson);
  }

  public Result getResult() {
```

LMK if that fixes the issue and does not break any other tests across the board.
Other option is to use assertThrows. But lets try the above fix first.

Reply:
yes worked,